index.html

```diff
@@ -0,0 +1,371 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <title>portfolio</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <div class="nav-container">
+            <div class="logo"><a href="#">logo <span>here</span></a></div>
+                <ul class="menu">
+                    <li><a href="#home" class="menu-btn">HOME</a></li>
+                    <li><a href="#about" class="menu-btn">ABOUT ME</a></li>
+                    <li><a href="#exped" class="menu-btn">EXPERIENCE</a></li>

```

Comment:
shall we add an additional two tabs Awards and Publications to add their special awards and research publications?